ng-test/src/app/shop.create/shop-create.component.html

```diff
@@ -50,7 +50,7 @@
       <label class="control-label">Postal Code</label>
       <input type="text" [ngModel]="shop?.address?.postalCode" (ngModelChange)="shop.address.postalCode = $event" name="postalCode" class="form-control" placeholder="postalCode" autofocus="autofocus" required="">
     </div>
-    <app-address-geo [(addressShop)]="shop.address" [addressReady]="addressR"  (addressReadyEvent)="addressR=$event"></app-address-geo>
+    <app-address-geo [(address)]="shop.address" (onAddressReady)="addressR=$event"></app-address-geo>
```

Comment:
tu ne peux pas directement placer `(onAddressReady)="shop.address=$event"` ?

Reply:
oui je peux mais alors là je peux pas passer l'address du composant parent a l'enfant, c'est pour ça que je bind 2 way l'address et que j'utilise ```onAddressReady``` pour informer si l'address est ok ou ap

Reply:
essayes avec une méthode à la place de la variable `updateAddress(address) {this.address=address}`